sim/core/agent.go

```diff
@@ -29,11 +26,6 @@ type Agent interface {
 	// will call this again; it will call Act() at the time specified by the return
 	// value.
 	Act(*Simulation) time.Duration
-
-	// This will be called instead of Act() for the very first loop iteration after each
-	// Simulation reset. Like Act(), the return value is the time at which the Agent
-	// would like Act() to be called.
-	Start(*Simulation) time.Duration
```

Comment:
this seems also like a good change, the fact that it was basically "act but different" was probably not an easy api to explain.